tooling/templatize/internal/end2end/e2e.go

```diff
@@ -34,14 +35,15 @@ import (
 var defaultRgName = "hcp-templatize"
 
 func shouldRunE2E() bool {
-	return os.Getenv("RUN_TEMPLATIZE_E2E") == "true"
+	// return os.Getenv("RUN_TEMPLATIZE_E2E") == "true"
```

Comment:
What's this all about?

Reply:
Running End to end test only, where it is required. I changed it , cause I wanted to debug the test and was to lazy to setup VSCode properly. :shaking_face: